### ISSUE TITLE
check_ping 2.2.1 bug?

### ISSUE THREAD

**Issue by jjani** [monitoring-plugins/monitoring-plugins#1606]
Hi!

In version 2.2.1 I tried modify these parameters lot of versions ("-w 100,5% -c 200,15% -4 -p 5 -t 30") when the host is denied any ICMP packet by iptables, but haven't got performance data in response, except with  "-p 1" parameter.
The answer was only: "CRITICAL - Plugin timed out"
This command working perfectly in version 2.1.4 and 2.1.2.
The answer was: "PING CRITICAL - Packet loss = 100%|rta=200.000000ms;100.000000;200.000000;0.000000 pl=100%;5;15;0"

Can you check it?
This is a bug or my error?

BR,
Janos Juhasz

**Comment by bardahlm** [comment]
This looks like the same bug I submitted to the nagios plugins project, https://github.com/nagios-plugins/nagios-plugins/issues/323

**Comment by sni** [comment]
i am confused, there is no 2.2.1 release here. What version are you using actually?

**Comment by RincewindsHat** [comment]
Can't reproduce this error, could this be something in nagios-plugins?
In that case, this issue should be closed.

**Comment by RincewindsHat** [comment]
Close this for lack of feedback. Reopen is the problem persists :-)
